### PR TITLE
Admin uploader

### PR DESCRIPTION
**Pull request by anandkanav**

### CODE REVIEW

**Comment by live-wire** [comment]
Can't find the uploader-option when I click on Add-People?
What the mother fuck bro ?

File: views/modal/AddPeopleModal.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+											<br/>
+											<button type="submit" ng-click="inviteIndividual()" class="btn btn-primary" ng-disabled="!(myForm.emailHolder.$valid && (checked == 'teacher' || checked == 'student'))">Invite</button>
+										</div>
+									</form>
+
+								</div>
+								<div class="row">
+									<div class="col-md-12">
+										<pre style="text-align: center;">
+											<label>Many people to invite? <button class="file-upload btn btn-success" >
```
Comment (anandkanav) [inline views/modal/AddPeopleModal.html:42]: what is this then?Are you looking closely?-.-

**Comment by live-wire** [comment]
screen shot 2016-07-15 at 11 30 29 am
I sure am -_- 
I just took your latest changes!

**Comment by anandkanav** [comment]
screen shot 2016-07-15 at 3 39 08 pm

**Pull request merged**